lightning-rapid-gossip-sync/src/processing.rs

```diff
@@ -526,6 +557,115 @@ mod tests {
 		assert!(after.contains("783241506229452801"));
 	}
 
+	#[test]
+	fn full_update_succeeds_with_edge_timestamp_age() {
+		const LATEST_SEEN_TIMESTAMP: u64 = 1642291930;
```

Comment:
Is the value chosen here significant?

Reply:
yes. It is the timestamp encoded in the input bytes.